config/locales/en.yml

```diff
@@ -81,6 +81,12 @@ en:
   invalid_credentials: Login failed due to invalid credentials. Are you sure you entered them correctly?
   invalid_login_method: Login failed due to account mismatch. You need to log in with omniauth.
   invite_message: "To invite someone to the meeting, send them this link:"
+  javascript:
+    room:
+      mailer:
+        subject: 'has invited you to view a recording.'
+        body: 'To view the recording, follow the link below:\n\n'
+        footer: '\n\nThis e-mail is auto-generated by BigBlueButton.\nBigBlueButton is an open source web conferencing system. For more information on BigBlueButton, see https://bigbluebutton.org/.'
```

Comment:
Same as above. You might need to split these into 2 separate entries

Reply:
Agreed